www-client/dillo/dillo-9999.ebuild

```diff
@@ -55,9 +58,10 @@ src_install() {
 	dodir /etc
```

Comment:
why did you keep this?

Reply:
Ahh, yes.. i actually wanted to ask about that.
I wasn't sure why it was there so i keep it, even though i already removed it in the first place. (but reverted back)
I've removed it now, thanks for the hint.